src/index.js

```diff
@@ -0,0 +1,85 @@
+// Importing dependancies
+const inquirer = require("inquirer");
+const path = require("path");
+const fs = require("fs");
+const { writeToFile } = require("./utils/utils.js");
+const generateHTMLfile = require("./utils/renderHtml.js");
+
+// Importing Classes
+const { name, id, email } = require("./lib/Employee");
+const Manager = require("./lib/Manager");
+const Engineer = require("./lib/Engineer");
+const Intern = require("./lib/Intern");
+
+// Importing Questions
+const {
+	introQuestions,
+	questions,
+	loopingQuestion,
+} = require("./questions.js");
+
+// Checking to see if the directory and file foor the generated HTML page already exists
+const OUTPUT_DIR = path.resolve(__dirname, "./src/dist");
+// If not, create directory and file
+const outputPath = path.join(OUTPUT_DIR, "team.html");
+
+// initialize function -prepares the application for user interaction and inputs
+const init = async () => {
+	let inProgress = true;
+
+	// Declaration of empty arrays to hold objects for each team member
+	let engineers = [];
+	let interns = [];
+	let managers = [];
+
+	// Initial question prompts - requesting team name and manager information from the user
+	const { name, id, email, officeNumber, teamName } = await inquirer.prompt(
+		introQuestions
+	);
+	// Instantiates a new instance of the Manager class and populates information based on the users inputs
+	const manager = new Manager(name, id, email, officeNumber, teamName);
+	// Pushes team member object to the managers array
+	managers.push(manager);
+
+	while (inProgress) {
+		const answers = await inquirer.prompt(questions);
```

Comment:
It would be good to destructure the object here, as you have on line 36 with the introQuestions.